README.md

```diff
@@ -131,13 +131,11 @@ module.exports = {
       },
     ],
   },
-  stats: {
-    warningsFilter: [/Failed to parse source map/],
-  },
+  ignoreWarnings: [/Failed to parse source map/],
```

Comment:
`ignoreWarnings` should be in `stats` options

Reply:
Not according to the output of Webpack. See https://github.com/webpack/webpack.js.org/issues/3995#issuecomment-710895415

Reply:
https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json#L3692, yep, you are right